basic_system/src/system_implementation/flat_storage_model/account_cache_entry.rs

```diff
@@ -237,8 +253,8 @@ impl AccountProperties {
                     + 8 // versioning data
                     + ValueDiffCompressionStrategy::optimal_compression_length_u256(initial.nonce.try_into().map_err(|_| internal_error!("u64 into U256"))?, r#final.nonce.try_into().map_err(|_| internal_error!("u64 into U256"))?) as u32 // nonce diff
                     + ValueDiffCompressionStrategy::optimal_compression_length_u256(initial.balance, r#final.balance) as u32 // balance diff
-                    + 4 // bytecode len
-                    + r#final.bytecode_len // bytecode
+                    + 4 // unpadded code len
```

Comment:
We need to change the order in compression. Before that we had full bytecode len and next `bytecode_len` contained actual bytecode. Now it's not a case, we need probably to have unpadded code len, artifacts len and then full bytecode

Reply:
I mean otherwise it's not possible to decode it

Reply:
Nice catch, fixed in latest commit